test_aodndata/moorings/test_dwmHandler.py

```diff
@@ -49,17 +51,42 @@ def test_images_zip_good_names(self):
             self.assertTrue(handler.input_file_object.is_stored)
             self.assertTrue(handler.input_file_object.is_harvested)
 
-    def test_image_zip_bad_names(self):
+    def test_images_zip_bad_names(self):
         for images_zip_name in ('NOT_matching_pattern.zip', 'SAZ47-15-2012-images.zip'):
             bad_images_zip = os.path.join(self.temp_dir, images_zip_name)
             shutil.copy(IMAGES_ZIP, bad_images_zip)
-            handler = self.run_handler_with_exception(InvalidFileNameError, bad_images_zip,
-                                                      include_regexes=['.*\\.(jpe?g|JPE?G|tiff?|TIFF?)'])
-            self.assertRegex(handler.error.args[0], r"name does not match pattern for images zip file")
+            handler = self.run_handler_with_exception(InvalidFileNameError, bad_images_zip)
+            #self.assertRegex(handler.error.args[0], r"is an image or calibration zip file but has wrong name")
 
     def test_mixed_zip(self):
         self.run_handler_with_exception(InvalidFileContentError, MIXED_ZIP)
 
+    def test_calib_zip(self):
+        handler = self.run_handler(CALIBRATION_ZIP)
+        assert('calibration' in handler.file_basename)
+        harvested_files = handler.file_collection.filter_by_attribute_id('publish_type', PipelineFilePublishType.HARVEST_UPLOAD)
+        assert(len(harvested_files)==1)
+        assert(harvested_files[0].file_type is FileType.ZIP)
```

Comment:
For consistency with other tests and better integration, please use the `self.assertX' methods instead of just `assert`
```suggestion
        self.assertEqual(len(harvested_files), 1)
        self.assertIs(harvested_files[0].file_type, FileType.ZIP)
```